openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/TopBar.vue

```diff
@@ -22,18 +22,23 @@
 
 <template>
   <mounting-portal mount-to="#openc3-menu" append>
-    <div class="v-toolbar__content">
+    <v-row no-gutters
+      ><v-col align-self="end">
+        <span class="app-title mr-2">{{ title }}</span></v-col
+      >
+    </v-row>
+    <v-row dense class="flex-nowrap">
```

Comment:
`flex-nowrap` key to making the title bar just push right ... it goes off the screen at some point but still better than the weird wrapping